.github/workflows/acceptance-tests.yml

```diff
@@ -1,8 +1,10 @@
 name: 'Acceptance Tests'
 
 on:
-  workflow_dispatch: {}
-  workflow_call: 
+  workflow_dispatch: {} # workflow can be run manually
+  schedule:
+    - cron: "0 4 * * *" # workflow runs every day at 04:00 AM
+  workflow_call: # workflow runs after code-health
```

Comment:
Updated the workflow to also run all the tests against cloud-dev once per day at 4 AM